.github/workflows/main.yml

```diff
@@ -75,3 +75,35 @@ jobs:
         with:
           command: clippy
           args: -- -D warnings
+
+  fuzz:
+    name: Fuzz Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust:
+          - nightly
```

Comment:
Hongfuzz frequently breaks on nightly, so all repos in the rust-bitcoin ecosystem switched to stable for fuzzing. The CI also fails to download dependencies in nghtly here. I think we should also change this to stable